turbo/jsonrpc/eth_receipts.go

```diff
@@ -91,10 +95,6 @@ func (api *APIImpl) GetLogs(ctx context.Context, crit filters.FilterCriteria) (t
 
 	if crit.BlockHash != nil {
 		num := rawdb.ReadHeaderNumber(tx, *crit.BlockHash)
-		//header, err := api._blockReader.HeaderByHash(ctx, tx, *crit.BlockHash)
```

Comment:
Better use “blockReader” - it can reqd data from snapshots

Reply:
Should i use blockReader` for fetching all blocks, or just in case when `BlockHash` is given to me?

Reply:
db storing only last 90K blocks - everything else is in snapshots